[OSDOCS-4464]: ccoctl gcp delete specify dir param (#52380)
openshift/openshift-docs: modules/cco-ccoctl-deleting-sts-resources.adoc

+
[source,terminal]
----
$ ccoctl aws delete \
  --name=<name> \ <1>
  --region=<aws_region> <2>
----
+
<1> `<name>` matches the name used to originally create and tag the cloud resources.

> <2> `<aws_region>` is the AWS region in which cloud resources will be deleted.

Reviewer: Nit: use present tense where possible. 

```suggestion
<2> `<aws_region>` is the AWS region where the cloud resources are deleted from.
```

+
[source,terminal]
----
$ ccoctl gcp delete \
--name=<name> \
--project=<gcp_project_id> \
--credentials-requests-dir=<path_to_directory_with_list_of_credentials_requests>
----
+
where:
+
** `<name>` matches the name used to originally create and tag the cloud resources.
** `<gcp_project_id>` is the GCP project ID in which cloud resources will be deleted.

> ** `<path_to_directory_with_list_of_credentials_requests>/credrequests` is the directory containing the files for the component `CredentialsRequest` objects.

Reviewer 2: There's a good chance that the `CredentialsRequests` from installation won't be available so I think we should link to how to extract them for $version from the installation steps.

2021/04/08 17:51:39 Identity Provider with ARN arn:aws:iam::<aws_account_id>:oidc-provider/<name>-oidc.s3.<aws_region>.amazonaws.com deleted
----
//Would love a GCP version of the above output.
endif::aws-sts[]
ifdef::google-cloud-platform[]
. Extract the list of `CredentialsRequest` custom resources (CRs) from the {product-title} release image by running the following command:
+
[source,terminal]
----
$ oc adm release extract --credentials-requests \
  --cloud=gcp \

>   --to=<path_to_directory_with_list_of_credentials_requests>/credrequests \ <1>

Author: It is long, but unfortunately it is this way on purpose due to the possibility of user choices when they set this up

  quay.io/<path_to>/ocp-release:<version>
----
+

> <1> `credrequests` is the directory where the list of `CredentialsRequest` objects is stored. This command creates the directory if it does not exist.

Reviewer: ```suggestion
<1> `<credentials_requests_dir>` is the directory where the list of `CredentialsRequest` objects is stored. This command creates the directory if it does not exist.
```

. Delete the GCP resources that `ccoctl` created:
+
[source,terminal]
----
$ ccoctl gcp delete \
  --name=<name> \ <1>
  --project=<gcp_project_id> \ <2>

>   --credentials-requests-dir=<path_to_directory_with_list_of_credentials_requests>/credrequests

Reviewer: Same comment as above. `<path_to_directory_with_list_of_credentials_requests>/credrequests` seems unnecessarily verbose.

```suggestion
  --credentials-requests-dir=<credentials_requests_dir>
```

----
+

> <1> `<name>` matches the name used to originally create and tag the cloud resources.

Reviewer: ```suggestion
<1> `<name>` matches the name that was originally used to create and tag the cloud resources.
```
